src/styles/main.scss

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
 }
+
+$cell-size: 100px;
+$day-margin: 0.5px;
+$container-columns: 7;
+$column-gap: 1px;
+$container-padding: 10px;
+$container-width: ($cell-size * $container-columns) + $column-gap *
+  $container-columns + ($container-padding * 2);
+
+$days-of-week: (
+  mon: 1,
+  tue: 2,
+  wed: 3,
+  thu: 4,
+  fri: 5,
+  sat: 6,
+  sun: 7,
+);
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $container-width;
+  padding: $container-padding;
+  box-sizing: border-box;
```

Comment:
Not fixed. Move the box-sizing to the * selector to ensure a consistent box model throughout your page